common/main/ks.xml

```diff
@@ -4307,7 +4307,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 					<pattern type="100000000000000" count="other" draft="provisional">↑↑↑</pattern>
 				</currencyFormat>
 			</currencyFormatLength>
-			<unitPattern count="one">{0} {1}</unitPattern>
+			<unitPattern count="one">↑↑↑</unitPattern>
```

Comment:
@srl295 This change was the result of running the DisplayAndInputProcessor. Seems to be marking inheritance; should I remove this change, or is it fine here?